packages/api-specification/src/new-window-api.html

```diff
@@ -60,7 +64,12 @@
     <dt>name</dt>
     <dd>Name of the new window. Should not contain whitespace</dd>
     <dt>features (optional)</dt>
-    <dd>DOMString to set features of the new window</dd>
```

Comment:
I know one of the principles is to follow the HTML5 API, but I think an object, rather than a string, would be much nicer for the consumer.

For example,  Electron's [BrowserWindow](https://github.com/electron/electron/blob/master/docs/api/browser-window.md#new-browserwindowoptions) has more than 40 options (nearly 70 if we include the options in `webPreferences`), I think building/maintaining a string with more than a handful of options could be pretty painful for the user.

```js
ssf.window(url, windowName, `child=${isChild},height=${height},width=${width},x=${x},y=${y}...`);
```

I think this is more of a discussion point, than something to fix, at the moment - but maybe having an API like this would work well:

```js
// options probably required, successCallback and errorCallback optional
new ssf.Window(options, successCallback, errorCallback);

// example usage
new ssf.Window({
  child: isChild,
  height,
  width,
  x,
  y
});

```

What do you think @BenLambertNcl? (cc @ColinEberhardt)